metastore/src/test/org/apache/hadoop/hive/metastore/InjectableBehaviourObjectStore.java

```diff
@@ -78,6 +84,22 @@ public static void resetListPartitionNamesBehaviour(){
     return listPartitionNamesModifier;
   }
 
+  // Methods to set/get/reset getNextNotification modifier
+  public static void setGetNextNotificationBehaviour(
+          com.google.common.base.Function<NotificationEventResponse,NotificationEventResponse> modifier){
+    getNextNotificationModifier = (modifier == null)? com.google.common.base.Functions.identity() : modifier;
+  }
+
+  public static void resetGetNextNotificationBehaviour(){
+    setGetNextNotificationBehaviour(null);
+  }
+
+  public static com.google.common.base.Function<NotificationEventResponse,NotificationEventResponse>
```

Comment:
The get methods are just there to compliment the set methods and may be needed in future. It can be removed as well.